bundle/wavefront-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,293 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "wavefront.com/v1alpha1",
+          "kind": "WavefrontProxy",
+          "metadata": {
+            "name": "example-wavefrontproxy",
+            "namespace": "wavefront"
+          },
+          "spec": {
+            "token": "\u003cTOKEN\u003e",
+            "url": "\u003chttps://YOUR_CLUSTER.wavefront.com/api/\u003e",
+            "image": "quay.io/wavefront/wavefront-proxy:5.7-rc1",
+            "openshift": true,
+            "storageClaimName": "wavefront-proxy-storage",
+            "enableAutoUpgrade": false
+          }
+        },
+        {
+          "apiVersion": "wavefront.com/v1alpha1",
+          "kind": "WavefrontCollector",
+          "metadata": {
+            "name": "wavefront-collector",
+            "namespace": "wavefront"
+          },
+          "spec": {
+            "daemon": true,
+            "openshiftDefault": true,
+            "enableAutoUpgrade": false, 
+            "env": [
+              {
+                "name": "HOST_PROC",
+                "value": "/host/proc"
+              },
+              {
+                "name": "POD_NODE_NAME",
+                "valueFrom": {
+                  "fieldRef": {
+                    "apiVersion": "v1",
+                    "fieldPath": "spec.nodeName"
+                  }
+                }
+              },
+              {
+                "name": "POD_NAMESPACE_NAME",
+                "valueFrom": {
+                  "fieldRef": {
+                    "apiVersion": "v1",
+                    "fieldPath": "metadata.namespace"
+                  }
+                }
+              }
+            ],
+            "image": "quay.io/wavefront/wavefront-kubernetes-collector:1.0.3-rc1",
+            "tolerations": [
+              {
+                "effect": "NoSchedule",
+                "key": "node.alpha.kubernetes.io/role",
+                "operator": "Exists"
+              },
+              {
+                "effect": "NoSchedule",
+                "key": "node-role.kubernetes.io/master",
+                "operator": "Exists"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Basic Install
```

Comment:
what are the allowed capabilities ? 
We support minor auto upgrades as well now.

Reply:
I think we support Seamless Upgrades based on description here - https://github.com/operator-framework/operator-sdk/blob/master/doc/images/operator-capability-level.png

Reply:
I guess current operator code support auto upgrades only for docker hub images.